src/User32.Desktop/User32+SpecialWindowHandles.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="SpecialWindowHandles"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        /// <summary>
+        /// Special window handles
+        /// </summary>
+        public enum SpecialWindowHandles
```

Comment:
I like the idea of having it as `IntPtr` (actually I was using it as constants this way before but I didn't want to pollute the class with countless constants so that's why I did it as an `enum`).

I will transform it on a static class.